src/mprismanager.cpp

```diff
@@ -0,0 +1,600 @@
+// -*- c++ -*-
+
+/*!
+ *
+ * Copyright (C) 2015 Jolla Ltd.
+ *
+ * Contact: Valerio Valerio <valerio.valerio@jolla.com>
+ * Author: Andres Gomez <andres.gomez@jolla.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+
+#include "mprismanager.h"
+
+#include "mpriscontroller.h"
+
+#include <qqmlinfo.h>
+
+#include <QDBusConnection>
+#include <QDBusConnectionInterface>
+
+#include <QtCore/QSignalMapper>
+
+static const QString mprisNameSpace = QStringLiteral("org.mpris.MediaPlayer2.*");
+static const QString dBusService = QStringLiteral("org.freedesktop.DBus");
+static const QString dBusObjectPath = QStringLiteral("/org/freedesktop/DBus");
+static const QString dBusInterface = QStringLiteral("org.freedesktop.DBus");
+static const QString dBusNameOwnerChangedSignal = QStringLiteral("NameOwnerChanged");
+
+
+MprisManager::MprisManager(QObject *parent)
+    : QObject(parent)
+    , m_singleService(false)
+    , m_currentController()
+    , m_availableControllers()
+    , m_otherPlayingControllers()
```

Comment:
In that case no need to add them here IMO, but I'm ok if you prefer this way, just raises some eyes :) in case they are not initialized is a potential risk, if you use some code check tool it will complain about it.